protocol/communities/community.go

```diff
@@ -836,11 +836,25 @@ func (o *Community) RemoveUserFromOrg(pk *ecdsa.PublicKey) (*protobuf.CommunityD
 func (o *Community) AddCommunityTokensMetadata(token *protobuf.CommunityTokenMetadata) (*protobuf.CommunityDescription, error) {
 	o.mutex.Lock()
 	defer o.mutex.Unlock()
-	if !o.IsControlNode() {
-		return nil, ErrNotControlNode
+
+	isControlNode := o.IsControlNode()
+	canManageTokens := o.IsOwnerWithoutCommunityKey() || o.IsTokenMaster()
```

Comment:
This name might be confusing, because in theory, the controlNode can also manage tokens. I'm not sure what else we can name it though.

One option is to change this to
```go
canManageTokens := o.IsControlNode() || o.IsOwnerWithoutCommunityKey() || o.IsTokenMaster()
``` 

Then on line 846, you can change that condition to `!isControlNode` instead.

The condition on line 842 can change to only `!canManageTokens` as well. 

What do you think?

Reply:
I would like to have `isControlNode` in a separate variable, as it was done for all other community "actions". Mixing will complicate the readiness of the code
I can rename it to `allowedToSendEvents` as it is on other community actions, or some other name :)

Reply:
I think `allowedToSendEvents` is slightly better, but in theory, the control ndoe can also send events if it wanted to.

Anyway, as long as everyone understands the code. What is you guys' preference? cc @osmaczko @0x-r4bbit

Reply:
To be hones, I don't think that control node should have a possibility to send community events, he can edit CommunityDescription directly

Reply:
Yeah, ok that makes sense. We can call it `allowedToSendEvents` then

Reply:
I would opt to be as explicit as possible -> `canSendTokenEvents`/`allowedToSendTokenEvents`. `allowedToSendEvents` is too generic, as an example: admins can send events but can't send token events.